src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
```

Comment:
rename - 
![image](https://github.com/Oleksiy-sovyak/my_react_dynamic-list-of-posts_test-task/assets/58264776/988d0a43-dc40-4bf3-907f-bd6b190c3f01)